### DISCUSSION TITLE
`loading workspace lua_ls` and `# Plugin Updates` on every nvim start

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [LazyVim/LazyVim#582] [Q&A] [answered]
I am not sure that this is related exactly to LazyVim.

Every time I start nvim, it get two progress bars with `loading workspace lua_ls`. Is it possible to do this only once?

Also, from **LazyVim** I get an annoying `# Plugin Updates` pop up on every start. Is the a way to disable this pop up?

**Comment by dpetka2001** [comment]
I don't know about your first question but for your second one regarding `Plugin Updates` you can configure it from lazy.nvim configuration, see [here](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration). It's under `checker.notify = true`. Change that to false and you shouldn't be getting any more notifications about plugin updates.

**Reply by dpetka2001** [reply]
The file you need to edit is described in the Lazyvim [docs](https://www.lazyvim.org/configuration/lazy.nvim).

**Comment by folke** [comment, marked as answer]
As @dpetka2001 says and those two loading bars for lua_ls are normal. Check the lua_ls wiki
